resources/var/www/html/errors/starting.html

```diff
@@ -1,29 +1,31 @@
 <!DOCTYPE html>
 <html>
 <head>
-	<meta charset="utf-8">
-	<meta name="viewport" content="width=device-width">
-	<meta http-equiv="refresh" content="10" />
-	<title>Dogu is starting</title>
-	<!--Stylesheet CES Theme-->
-	<link rel="stylesheet" href="/errors/css/ces.css" media="screen">
-	<link rel="stylesheet" href="/errors/css/dogu-starting-styles.css">
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1"/>
+    <meta http-equiv="refresh" content="10" />
+    <title>Dogu startet</title>
+    <link rel="stylesheet" href="/errors/css/dogu-starting.css">
 </head>
 <body>
-<div class="logo">
-	<img src="/errors/img/logo-white-320px.png">
-</div>
-<div class="container">
-	<div class="row">
-		<div class="col-xs-6">
-			<div class="imageContainer flyIn">
-				<img class="flyingCloud " src="/errors/img/flying_cloud.svg">
-			</div>
-		</div>
-		<div class="col-xs-8 col-sm-6 starting">
-			<h1 id="doguName">Dogu is starting</h1>
-		</div>
-	</div>
+<div class="dogu-starting">
+    <div>
+        <img alt="" class="logo">
```

Comment:
This image element is not properly closed. Don't we have an HTML validator at hand to catch this?